README.md

```diff
@@ -1,76 +1,43 @@
-| `pydemolib/` | Python library that helps to eliminate nearly all boilerplate code in Jupyter notebook based demos. This is not meant to be used in production. See also the disclaimer [here](pydemolib/README.rst).
-| `configs/` | Configuration files that define Nessie, Iceberg, Spark versions and dependencies, supporting demos using different product versions.
-| `datasets/` | Various datasets that can be used by the demos. The nessiedemo Python library in `pydemolib/` supports downloading and accessing these datasets.
-| `/` | Also contains `apt.txt` + `requirements.txt` used by Binder. This `requirements.txt` has nothing to do with the `requirements.txt` in `pydemolib/`.
+### Nessie
 
-## How to add a new Jupyter Notebook Demo?
+Nessie version is set in Binder at `binder/requirements.txt` and for unit tests in `notebooks/tox.ini`. Currently, Iceberg and Delta
+both support only 0.9.x of Nessie.
 
-A new demo will generally require changes in the following places:
-* adding a new notebook under `colab`
-* adding a new config file with the required demo dependencies under `configs`. This new config file is then referenced in `setup_demo("my_new_config.yml", ["nba"])`
-* if the new demo uses a new dataset, then this dataset needs to be added to `datasets` and properly referenced in the notebook: `setup_demo("my_new_config.yml", ["my_new_dataset"])`
-* a new unit test under `notebook-tests/tests`
-* if the new demo requires some dependency setup code that isn't already covered in `pydemolib/nessiedemo`, it will then require a new module under `pydemolib/nessiedemo` with additional tests under `pydemolib/tests`
-The entire demo setup code for the new notebook would then look as shown below:
+### Iceberg
 
-```python
-from nessiedemo.demo import setup_demo
-demo = setup_demo("my_new_config.yml", ["my_new_dataset"])
+Currently we are using Iceberg `0.12.0` and it is specified in both iceberg notebooks as well as `notebooks/tests/__init__.py`
 
-# Downloads and sets up X
-from nessiedemo.new_module_x import x_for_demo
-x = x_for_demo(demo)
-```
+### Delta
 
-## Jupyter notebook environments
+currently Delta version is taken directly from the Nessie version and isn't explicitly noted. It is currently `1.0.0-nessie`
 
-### Google Colaboratory
+### Spark
 
-[Google Colaboratory](https://colab.research.google.com/) provides a Jupyter environment.
-Users basically just provide a URL to a `.ipynb` file, which is then loaded. The runtime
-environment is a container/image that is pre-defined by Google. Changing the base operating
-system and such is not possible. Bumping the pre-installed Python dependencies is not feasible.
+Only has to be updated in `binder/requirements.txt`. Currently Iceberg supports 3.0.x and 3.1.x while delta late supports
+3.1.x only.
 
-### Binder
+### Flink
 
-[Binder](https://mybinder.org) is a more customizable platform for Jupyter notebooks and
```

Comment:
Do you think if it makes sense to keep the brief description of Binder for anyone who is not familiar with it (including myself) :D ?

Reply:
done